test/versioned/aws-sdk-v3/package.json

```diff
@@ -10,7 +10,8 @@
     {"name": "@aws-sdk/lib-dynamodb", "minAgentVersion": "8.7.1"},
     {"name": "@aws-sdk/smithy-client", "minSupported": "3.47.0", "minAgentVersion": "8.7.1"},
     {"name": "@smithy/smithy-client", "minSupported": "2.0.0", "minAgentVersion": "11.0.0"},
-    {"name": "@aws-sdk/client-bedrock-runtime", "minAgentVersion": "11.13.0"}
+    {"name": "@aws-sdk/client-bedrock-runtime", "minAgentVersion": "11.13.0"},
+    {"name": "@aws-sdk/client-redshift-data", "minAgentVersion": "12.10.0"}
```

Comment:
this will need to be whatever version it will be released in. probably `12.12.0`